src/styles/main/main-block.scss

```diff
@@ -0,0 +1,20 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 0 73px;
+
+  &__row {
+    display: flex;
+    flex-grow: 1;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 944px;
+    padding-bottom: 46px;
+    margin: 0 auto;
+
+    &:last-child {
+      padding-bottom: 0;
+    }
+
+  }
+}
```

Comment:
![Screenshot_2](https://user-images.githubusercontent.com/121685347/221919632-4d2cd97e-99fc-4a63-9b84-dffb5d930fb9.png)
The assignment is a little different, please correct it.
You should only have four blocks in one row. No more, no less, just four